pkg/generated/openapi/def.bzl

```diff
@@ -38,15 +38,17 @@ def openapi_library(name, tags, srcs, go_prefix, vendor_prefix = "", openapi_tar
         cmd = " ".join([
             "ORIG_WD=$$(pwd);",
             "cd $$GOPATH/src/" + go_prefix + ";",
-            "$$ORIG_WD/$(location //vendor/k8s.io/code-generator/cmd/openapi-gen)",
+            "$$ORIG_WD/$(location //vendor/k8s.io/kube-openapi/cmd/openapi-gen)",
             "--v 1",
             "--logtostderr",
             "--go-header-file $$ORIG_WD/$(location //" + vendor_prefix + "hack/boilerplate:boilerplate.go.txt)",
             "--output-file-base zz_generated.openapi",
             "--output-package " + go_prefix + "pkg/generated/openapi",
+            "--report-filename tmp_api_violations.report",
             "--input-dirs " + ",".join([go_prefix + target for target in openapi_targets] + [go_prefix + "vendor/" + target for target in vendor_targets]),
             "&& cp $$GOPATH/src/" + go_prefix + "pkg/generated/openapi/zz_generated.openapi.go $$ORIG_WD/$(location :zz_generated.openapi.go)",
+            "&& rm tmp_api_violations.report",
         ]),
         go_deps = deps,
-        tools = ["//vendor/k8s.io/code-generator/cmd/openapi-gen"],
```

Comment:
we should also remove openapi-gen from code-generator repo after this PR got merged.

Reply:
Sounds good